Tests/LibPDF/CMakeLists.txt

```diff
@@ -5,3 +5,6 @@ set(TEST_SOURCES
 foreach(source IN LISTS TEST_SOURCES)
     serenity_test("${source}" LibPDF LIBS LibCore LibPDF)
 endforeach()
+
+file(GLOB PDF_FILES CONFIGURE_DEPENDS "../../Base/home/anon/Documents/pdf/*.pdf")
+install(FILES ${PDF_FILES} DESTINATION usr/Tests/LibPDF)
```

Comment:
Err, what? No, why would we install the files twice? (Once in /home/anon/, and again in /usr/Tests/LibPDF.) Also, so far we only have executables in /usr/Tests/ I believe. If you can find a way to keep it this way, I'd prefer it.

Reply:
https://github.com/SerenityOS/serenity/pull/10969#issuecomment-974692324
This is based on my interpretation of a comment that @ADKaster made.

Also I cannot confirm that there are only executables in `/usr/Tests/`. There are some exceptions already. 

- `/usr/Tests/LibWebtokenizer-test.html`
- `/usr/Tests/LibCore{long_lines.txt,10kb.txt}`

However if that is not the right place for such files I am happy to move them somewhere else. More feedback is appreciated.

Reply:
I think the test files being in/usr/Tests is fine until we come up with some kind of common Test input directory. Not sure about the request to not have them duplicated, are you saying we should remove them from Base/ and put them in Tests/LibPDF instead on the repo side, Ben?

Reply:
I take it back then, and whereever you put it is fine with me ^^